core/src/validator.rs

```diff
@@ -168,7 +173,36 @@ impl Validator {
             config.storage_slots_per_turn,
             bank.slots_per_segment(),
         );
-
+        let bf = bank_forks.clone();
```

Comment:
Can you wrap this entire new block in:
```rust
if config. enable_ctrl_c_handler {
..
}
```
since we only want to execute it when the handler is enabled.   

Once that's done within the new `if` block we can do this, which will avoid the need to use a short less readable `bf` variable name :
```diff
- let bf = bank_forks.clone();
+ let bank_forks = bank_forks.clone();
```

Reply:
Done